src/service/cia301/co_csdo.c

```diff
@@ -42,8 +36,8 @@ static CO_ERR COCSdoInitDownloadSegmented  (CO_CSDO *csdo);
 static CO_ERR COCSdoDownloadSegmented      (CO_CSDO *csdo);
 static CO_ERR COCSdoFinishDownloadSegmented(CO_CSDO *csdo);
 
-static CO_ERR COCSdoInitUploadBlock        (CO_CSDO *csdo);
-static CO_ERR COCSdoUploadSubBlock         (CO_CSDO *csdo);
+// static CO_ERR COCSdoInitUploadBlock        (CO_CSDO *csdo);
```

Comment:
recommend removal rather than commenting out

Reply:
There are additional instances of commented lines bellow

Reply:
done